web/index.html

```
@@ -961,13 +960,21 @@ <h1 style="margin:0;font-size:0">מפת העורף</h1>
         </button>
       </div>
       <div id="ext-menu">
-        <div id="menu-predict" class="menu-item f-gated">
+        <div id="menu-driving-mode-feature" class="menu-item">
```

HTMLHint (1.9.2)

[error] The id value [ menu-driving-mode-feature ] must be unique.

(id-unique)